src/main/java/com/openshift/restclient/model/IContainer.java

```diff
@@ -56,6 +58,12 @@
 	void setLifecycle(ILifecycle lifecycle);
 	ILifecycle getLifecycle();
 	
+    void setCommand(Set<String> command);
+    Set<String> getCommand();
+    
+    void setCommandArgs(Set<String> args);
```

Comment:
Might this better be implemented as a String... or a list since Set does not guarantee order.  Is order relevent?